ApiReview/API.Polly.Core/NoDocs/Polly.Strategy/Outcome.TResult.cs

```diff
@@ -0,0 +1,20 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.ExceptionServices;
+using System.Threading.Tasks;
+
+namespace Polly.Strategy;
+
+public readonly struct Outcome<TResult>
+{
+    public Exception? Exception { get; }
+    public TResult? Result { get; }
+    public bool HasResult { get; }
+    public bool IsVoidResult { get; }
+    public Outcome(Exception exception);
+    public Outcome(TResult result);
```

Comment:
Does this preclude someone having a successful outcome that is `null`? Should it be `TResult?`?